app/build.gradle

```diff
@@ -118,22 +118,22 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3"
 
-    implementation 'androidx.lifecycle:lifecycle-extensions:2.0.0'
+    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
 
-    implementation 'com.google.android.material:material:1.0.0'
-    implementation 'androidx.vectordrawable:vectordrawable:1.0.0'
+    implementation 'com.google.android.material:material:1.0.0' // mobile-ui lib conflicts w/1.1.0

```

Comment:
in trying to resolve issues, i tried to update material lib to 1.1.0 but the build bombs because there is a dependent lib in Sage that defines an attribute in attr.xml incorrectly.  that comment is left in there just as a warning to them.